src/OrchardCore.Modules/OrchardCore.Users/Drivers/UserDisplayDriver.cs

```diff
@@ -157,7 +191,16 @@ public override async Task<IDisplayResult> UpdateAsync(User user, UpdateEditorCo
 
                     foreach (var role in rolesToRemove)
                     {
-                        await _userRoleStore.RemoveFromRoleAsync(user, _userManager.NormalizeName(role), default(CancellationToken));
+                        // Make sure we always have at least one administrator account
+                        if (usersOfAdminRole.Count == 1 && usersOfAdminRole.First().UserName == user.UserName && role == _administratorRole)
+                        {
+                            _notifier.Warning(H["Cannot remove administrator role from the only administrator."]);
```

Comment:
Cannot remove administrator role from the ~~only~~ **super** administrator.

Reply:
In fact this message would be misleading as we can disable the first user account created on the backend as long as we log on a second administrator account. So we are not preventing deleting the "super administrator" account if you want to name it that way. If we should do ; then let's discuss about why.

Reply:
> we don't have a super administrator account.

What about the line 101 ;)

Reply:
Line 101 says. You can't disable the last administrator account remaining. Which could be a different user account than the one created on setup. This doesn't prevent disabled the "super administrator" account because we have never defined one. Do we have one?